### PR TITLE
Automations and return shipments

### PR DESCRIPTION
**Pull request by joosev**
- Create shipping labels automatically
- Aloow creation of multiple return labels to a order
- Refactoring

### CODE REVIEW

**Review by l3ku — COMMENTED** [review]
Did a series of quick test orders. Got this into my PHP error log:
```
[26-Aug-2019 10:00:50 UTC] PHP Fatal error:  Uncaught Error: Call to a member function __toString() on null in /data/wordpress/htdocs/wp-content/plugins/woo-pakettikauppa/includes/class-wc-pakettikauppa-admin.php:750
Stack trace:
#0 /data/wordpress/htdocs/wp-content/plugins/woo-pakettikauppa/includes/class-wc-pakettikauppa-admin.php(717): WC_Pakettikauppa_Admin->create_return_label(Object(WC_Order))
#1 /data/wordpress/htdocs/wp-content/plugins/woo-pakettikauppa/includes/class-wc-pakettikauppa-admin.php(77): WC_Pakettikauppa_Admin->save_ajax_metabox('370')
#2 /data/wordpress/htdocs/wordpress/wp-includes/class-wp-hook.php(286): WC_Pakettikauppa_Admin->ajax_meta_box('')
#3 /data/wordpress/htdocs/wordpress/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters('', Array)
#4 /data/wordpress/htdocs/wordpress/wp-includes/plugin.php(453): WP_Hook->do_action(Array)
#5 /data/wordpress/htdocs/wordpress/wp-admin/admin-ajax.php(99): do_action('wp_ajax_paketti...')
#6 {main}
  thrown in /data/wordpress/htdocs/wp-content/plugins/woo-pakettikauppa/includes/class-wc-pakettikauppa-admin.php on line 750
```

Functionality for the deletion of individual package slips has been implemented here by deleting WooCommerce comments, but this is not very clear at least to me because the label for it is "Poista kommentti". Also, the metabox "Poista pakettikortti" button is now confusing because it is possible for the order to have multiple slips, whereas the button was originally added at a stage where the plugin only supported creating a single slip to the order. By pressing "Poista pakettikortti" the meta box looks like there are no longer any slips created, even though there can be multiple. Perhaps the button could be removed and somehow the deletion action could be made more apparent from the comments by hooking into the WooCommerce? Also, maybe the deletion success message "Pakettikauppa-lähetyksen pakettikortti poistettiin onnistuneesti." could also include the slip ID for reference?
Screen Shot 2019-08-26 at 13 04 36

Code looks ok after quick skim through.

**Comment by joosev** [comment]
Return label creation should be visible like this. So creating multiple shipping labels in the future will become possible, but for now, only multiple return labels are possible.

Näyttökuva 2019-8-26 kello 15 10 59

**Comment by joosev** [comment]
Now it displays the tracking code of deleted shipping label

Näyttökuva 2019-8-26 kello 15 19 19
